server/user/tests/test_pk.py

```diff
@@ -42,6 +56,7 @@ def setUp(self):
     @parameterized.expand(
         [
             ("user2", status.HTTP_403_FORBIDDEN),
+            ("user3", status.HTTP_200_OK),
```

Comment:
```suggestion
            ("user3", status.HTTP_200_OK),
            ("user1", status.HTTP_200_OK),
```

not connected with your PR actually - just adds a test for `IsOwnUser` for better coverage